Cargo.lock

```diff
@@ -5256,7 +5256,7 @@ dependencies = [
  "chrono",
  "lazy_static",
  "matchers",
- "parking_lot 0.11.0",
+ "parking_lot 0.9.0",
```

Comment:
I don't think this change was intentional.

Reply:
It was not, I don't know how it popped up